stream/client.py

```diff
@@ -101,6 +111,10 @@ def get_full_url(self, relative_url):
         url = self.base_url + self.version + '/' + relative_url
         return url
 
+    def get_full_personal_url(self, relative_url):
+        url = self.base_url + '/personalization/' + relative_url + '/'
```

Comment:
URI only approach limits our options since we can't use DNS or simple LB to route traffic.

My suggestion is to keep `/personalization/` in the URI and to use a different subdomain.

eg.

Regular API end-point
https://us-east.stream-io-api.com/api/

Custom Personalization API end-point 
https://us-east-personalization.stream-io-api.com/personalization/

Custom API end-point
https://ua.stream-io-api.com/api/

Custom Personalization API end-point
https://ua-personalization.stream-io-api.com/personalization/

Reply:
@BalazsHoranyi this pull request looks good, can you implement the request from Tommaso to use a different DNS record for the personalization request? (not the meta one, just the personalization one). This will make it easier for our devops team to reroute this traffic in case there are issues.